Dockerfile

```
@@ -0,0 +1,16 @@
+FROM python:3.8.5-slim-buster as base
+RUN apt-get update
+RUN pip install "poetry==1.0.0"
+WORKDIR /do-me
+COPY poetry.lock poetry.toml pyproject.toml /do-me/
+RUN poetry install
+COPY ./todo_app/ /do-me/todo_app/
+EXPOSE 5000
+
+FROM base as production
+COPY ./run_gunicorn.sh ./run_gunicorn.sh
+ENTRYPOINT ["/bin/bash"]
```

Is there a reason you're just setting the entrypoint to bash? I assume we actually want it to run your gunicorn command, so we could set a default argument here like:

```suggestion
ENTRYPOINT ["/bin/bash"]
CMD ["run_gunicorn.sh"]
```

---

Thank you, well spotted